matRad_getRotationMatrix.m

```diff
@@ -56,24 +57,34 @@
         %active, counter-clockwise rotation Matrix for Gantry around z 
         %with pre-multiplication of the matrix (R*x)
         %Note: Gantry rotation is physically an active rotation of a beam 
-        %vector around the target / isocenterin the patient coordinate
+        %vector around the target / isocenter in the patient coordinate
         %system
         R_Gantry = [cosd(gantryAngle)  -sind(gantryAngle)  0; ...
             sind(gantryAngle)    cosd(gantryAngle)  0; ...
             0                            0           1];
         
+        %active, clockwise rotation for collimator around y
+        %with pre-multiplication of the matrix (R*x)
+        %Note: Collimator rotation is physically an active rotation of a beam 
+        %vector around the target / isocenter in the patient coordinate
+        %system  
+        R_Collimator = [cosd(-collimatorAngle) 0 sind(-collimatorAngle); ...
+            0 1 0; ...
+            -sind(-collimatorAngle)    0    cosd(-collimatorAngle)];
```

Comment:
I'll trust you on this one

Reply:
On our site this set up the right collimator angle.